extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/resources/SqlStatementResource.java

```diff
@@ -115,6 +117,7 @@
 public class SqlStatementResource
 {
 
+  public static final String RESULT_FORMAT = "resultFormat";
```

Comment:
I think we should set this as __resultFormat ie this is set by the broker and not by the user directly. Sort of like an internal parameter. 
WDYT ?

Reply:
Since this is a restricted parameter for the user, this should also be added here https://github.com/apache/druid/blob/2d8e0f28f36f83e091911cc7b36a57b10fd065cc/extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/MSQTaskSqlEngine.java#L58

Reply:
I think SYSTEM_CONTEXT_PARAMETERS are for those params set within MSQ itself. This is being set at the resource, so this should would always fail and the check wouldn't matter anyway since it is not set later in MSQ either.